test/uk/gov/hmrc/cdsreimbursementclaimfrontend/controllers/claims/EnterImporterEoriNumberControllerSpec.scala

```diff
@@ -44,22 +42,33 @@ class EnterImporterEoriNumberControllerSpec
     extends ControllerSpec
     with AuthSupport
     with SessionSupport
-    with ScalaCheckDrivenPropertyChecks {
+    with TableDrivenPropertyChecks
+    with ScalaCheckPropertyChecks {
 
   override val overrideBindings: List[GuiceableModule] =
     List[GuiceableModule](
       bind[AuthConnector].toInstance(mockAuthConnector),
       bind[SessionCache].toInstance(mockSessionCache)
     )
 
+  val testCases = Table(
+    "JourneyBindable",
+    JourneyBindable.Single,
+    JourneyBindable.Multiple,
+    JourneyBindable.Scheduled
+  )
```

Comment:
Shiva, just wondering if we using ScalaCheck then perhaps we don't want to mix it with TableDrivenTests?
I mean we generate random journey bindable. How do you think is it alright?